pkgs/top-level/all-packages.nix

```diff
@@ -29890,4 +29890,6 @@ in
   lc3tools = callPackage ../development/tools/lc3tools {};
 
   zktree = callPackage ../applications/misc/zktree {};
+
+  aseq2json = callPackage ../os-specific/linux/aseq2json {};
```

Comment:
Please sort this into a fitting category and sort there alphabetically.

Reply:
I was trying to figure out if that file is ordered; should have read the comment at the top instead :smile: